app/src/main/java/com/sdex/activityrunner/manifest/ManifestViewerActivity.kt

```diff
@@ -152,10 +155,21 @@ class ManifestViewerActivity : BaseActivity() {
                         (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK ==
                                 Configuration.UI_MODE_NIGHT_YES))
 
+    override fun onSaveInstanceState(outState: Bundle) {
```

Comment:
How do you think, is it possible to use `onSaveInstanceState/onRestoreInstanceState` directly in the `HighlightJsView` to encapsulate saving and restoring scroll position?